ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -227,6 +229,7 @@ public static KsqlRestApplication buildApplication(
              BrokerCompatibilityCheck.create(ksqlConfig.getKsqlStreamConfigProps(), topicClient)) {
       compatibilityCheck.checkCompatibility();
     }
+    final String kafkaClusterId = adminClient.describeCluster().clusterId().get();
```

Comment:
Where was this API added in the admin client? would it change our version compatibility matrix?

Reply:
0.10.1

Reply:
we'll compile against the latest admin client, but cluster id can be queried for any cluster 0.10.1 and later.